docs/developer/app-store/apps/np-atobarai/overview.mdx

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_label: Overview
+sidebar_position: 1
+title: NP Atobarai App Overview
+---
+
+import { AppMetadata } from "/components/AppMetadata/AppMetadata.jsx";
+
+<AppMetadata
+  minSaleorVersion="3.20"
+  githubUrl="https://github.com/saleor/apps/tree/main/apps/NP Atobarai"
+/>
+
+NP Atobarai is a Japanese payment gateway that supports payments on pick-up points.
+
+## Features
+
+- App integrates with NP Atobarai payment gateway.
+- App provides `PAYMENT_GATEWAY_INITIALIZE_SESSION` mutation to validate required data: shipping and billing addresses, phone number, currency, and email.
+
+
+## Limitations and assumptions
+
+Before installing this app, please review these limitations to ensure they don't conflict with your requirements.
+
+- App works only for channels with `JPY` currency.
+- App creates transactions within NP Atobarai during the fulfillment process, exactly when tracking number is updated. That means in case of invalid data, direct feedback on the storefront is not possible.
+- App works only on `CHARGE` flows. Saleor has limitation to allow fulfillments only when the order is fully charged.
+- When a customer does not pick up the delivery, the app will *not* report this to Saleor. Order will remain in the `FULFILLED` and `PAID` state. However, this should be safe, because NP Atobarai should not allow user to pick up the good without paying.
+- The App creates only one transaction in NP Atobarai and expects only one to exist for a single transaction in Saleor. If NP Atobarai returns multiple transactions, the app will report `CHARGE_FAILURE`
+- The App uses `transaction.token` as a identifier set in NP Atobarai, due to ID characters limit.
+- Tracking number set on fulfillment must have at most 20 characters.
+- Split payments are not supported. App work *only* if a single Atobarai transaction is created to the checkout/order.
+
+## Flow
+
+### Pre-requisites
+
+- You need to have a valid Japanese channel configured.
+- You need to have a Storefront that integrates with Transactions API (see docs).
+
+### Early validation
+
+Call `paymentGatewayInitialize` mutation to validate that the required data is present in a checkout.
+This mutation will return an error if any of the required data is missing, such as shipping and billing addresses, phone number, currency, or email.
+
+TODO: Error shape
+
+### Initialize payment
+
+Use `transactionInitialize` mutation to start the payment process. The app will perform a validation, and if data seems valid, it will call NP Atobarai to create the transaction.
```

Comment:
```suggestion
Use `transactionInitialize` mutation to start the payment process. The app will perform a validation, and if data seems valid, it will call NP Atobarai to register the transaction (and run credit check).
```